themes/doks/layouts/shortcodes/enterprise-icon.html

```diff
@@ -9,6 +9,6 @@
     background-color: transparent !important;
   }
 </style>
-<a {{ with .Get "noMargin" }} style="margin-left: 0 !important;" {{ end }} title="This feature is only available in the Enterprise edition of Corda." href="#detailedlinkgoeshere" class="enterprise-icon">
+<a {{ with .Get "noMargin" }} style="margin-left: 0 !important;" {{ end }} title="This feature is only available in the Enterprise edition of Corda." href="https://docs.r3.com/en/platform/corda/5.0/introduction/about-the-docs.html#documenting-product-editions" class="enterprise-icon">
   <img src="/icons/enterprise-icon.png" alt="enterprise-icon"  width="28px" />
```

Comment:
if you accept my previous comment - needs changing here

Reply:
This is not the same screenshot (.png) that I have renamed. This relates to the previous merge by Zheng. This is the class used for the actual icon that he created, so this one is ok.